pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -703,6 +735,63 @@ async def verify_attached_operators(self, *advisories: List[int], gather_depende
         _LOGGER.info("Running command: %s", cmd)
         await exectools.cmd_assert_async(cmd, env=self._elliott_env_vars, cwd=self.working_dir)
 
+    async def remove_builds_all(self, advisory_id):
```

Comment:
we can import elliottlib and use the function directly instead of invoke in subprocess

Reply:
Other functions in the file use the elliott command instead of importing, so went with the same pattern.

Reply:
other functions probably come from before elliott and pyartcd shared a repo; would be good to look for opportunities to reuse code directly.

but in doing so you also have to watch out for things that expect a lot of environment, e.g. the runtime. difficult to construct that without a complete invocation.